argcomplete/__init__.py

```diff
@@ -348,6 +343,30 @@ def collect_completions(self, active_parsers, parsed_args, cword_prefix, debug):
                             completions += subprocess.check_output(bashcomp_cmd).decode(sys_encoding).splitlines()
                         except subprocess.CalledProcessError:
                             pass
+
+            return completions
+
+        completions = []
+        """
+        for active_parser in active_parsers[:-1]:
+            debug('active_parser:', active_parser)
+            if self.always_complete_options or (len(cword_prefix) > 0 and cword_prefix[0] in active_parser.prefix_chars):
+                completions += get_option_completions(active_parser, cword_prefix)
+                debug('completions options:', completions)
+
+            completions = complete_active_option(active_parser, cword_prefix, parsed_args, completions)
+        """
```

Comment:
should we ignore words after cursor? cut it before split_line like this https://github.com/lisongmin/argcomplete/commit/f605e3a49ba993ac80aaca40bbe5f223457ccf13